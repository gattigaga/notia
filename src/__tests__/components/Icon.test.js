import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";

import Icon from "components/Icon";

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Icon name="list" />);
});

describe("Icon", () => {
  it("renders correctly", () => {
    expect(toJSON(wrapper)).toMatchSnapshot();
  });

  it("renders with additional class", () => {
    wrapper.setProps({ className: "my-class" });
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
