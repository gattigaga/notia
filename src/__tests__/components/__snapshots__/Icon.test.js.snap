// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon renders correctly 1`] = `
<i
  className="material-icons "
  onClick={[Function]}
>
  list
</i>
`;

exports[`Icon renders with additional class 1`] = `
<i
  className="material-icons my-class"
  onClick={[Function]}
>
  list
</i>
`;
